libs/application/templates/inheritance-report/src/lib/dataSchema.ts

```diff
@@ -35,7 +35,13 @@ const validateDeceasedShare = ({
 }
 
 const validateAssetNumber = (assetNumber: string) => {
-  const assetNumberPattern = /^(F\d{3}-\d{4}|\d{7}|\d{3}-\d{4}|F\d{7})$/
+  const assetNumberPattern =
+    /^(F\d{3}-\d{4}|\d{7}|\d{3}-\d{4}|F\d{7}|L\d{3}-\d{3}|L\d{6})$/
+  return assetNumberPattern.test(assetNumber)
```

Comment:
I'm wondering if we've started overthinking this?
Shouldn't we allow 6 or 7 numbers with L optional on 6 and F optional on 7?

I know the lookup client allows numbers exclusively so that shouldn't be a problem.

Something like: `/^[Ff]{0,1}\d{7}$|^[Ll]{0,1}\d{6}$/` maybe?